CMakeLists.txt

```diff
@@ -98,6 +98,16 @@ if (SMART_LINK)
   set(NEURON_BASE_SOURCES ${NEURON_BASE_SOURCES} src/connection/smart_link.c)
 endif()
 
+if (CLIB)
+  message(STATUS "set clib")
+  add_definitions(-DNEU_CLIB=${CLIB})
+endif()
+
+if (CLIB_VERSION)
```

Comment:
libc version is determined by the system and cannot be determined at compile time